### PR TITLE
Add hrishikeshnikam2000

### LINKED ISSUE #612: Hrishikesh Nikam
image

### PR DESCRIPTION
**Pull request by hrishikeshnikam2000**
resolve #612

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1169,4 +1169,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/28563357/126220805-82470ad3-f38a-40b4-a277-8c40704fdb5b.png",
     "issueNumber": 607
   }
+  {
+    "name": "Hrishikesh Nikam",
+    "githubUsername": "hrishikeshnikam2000",
+    "imageUrl": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
+    "issueNumber": 612
+}
```
Comment (schmelto) [inline data.json:1177]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 2]
```diff
@@ -1169,4 +1169,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/28563357/126220805-82470ad3-f38a-40b4-a277-8c40704fdb5b.png",
     "issueNumber": 607
   }
```
Comment (schmelto) [inline data.json:1171]:
```suggestion
   }, 
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
Sorry missed this one

File: data.json [hunk 3]
```diff
@@ -1168,5 +1168,11 @@
     "githubUsername": "MrSlimCoder",
     "imageUrl": "https://user-images.githubusercontent.com/28563357/126220805-82470ad3-f38a-40b4-a277-8c40704fdb5b.png",
     "issueNumber": 607
+   },
```
Comment (schmelto) [inline data.json:1171]:
```suggestion
  }, 
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
Lgtm

**Comment by schmelto** [comment]
Thanks for contributing and nice profile btw 🚀

**Comment by hrishikeshnikam2000** [comment]
Thank you so much! @schmelto

**Pull request merged**